README.md

```diff
@@ -48,14 +48,17 @@ Then, you can use the `FlaskPooledClusterRpcProxy` singleton just as you would n
 
 `FlaskPooledClusterRpcProxy` accepts all nameko configuration values, prefixed with the `NAMEKO_` prefix. In addition, it exposes additional configuration options:
 
-* `INITIAL_CONNECTIONS (int, default=2)` - the number of initial connections to the Nameko cluster to create
-* `MAX_CONNECTIONS (int, default=8)` - the max number of connections to the Nameko cluster to create before raises an error
+* `NAMEKO_INITIAL_CONNECTIONS (int, default=2)` - the number of initial connections to the Nameko cluster to create
+* `NAMEKO_MAX_CONNECTIONS (int, default=8)` - the max number of connections to the Nameko cluster to create before raises an error
+* `NAMEKO_LAZY_LOAD_SERVICCES (bool, default=False)` - whether connections to services should be loaded when the service is accessed (False) or when a method is called on a service (True)
```

Comment:
Also, is there any reason that setting this to `True` _shouldn't_ be the default behavior? It seems like we would always want it to need a connection as late as possible, no?